.gitignore

```diff
@@ -31,6 +31,7 @@ test/wasm-binaries-*.tbz2
 test/wasm-torture-s-*.tbz2
 test/wasm-install/
 test/validator/*.wasm
+*.map
```

Comment:
Is this really needed?  Is the build or test process dropping .map files in the source tree?

Reply:
I'm afraid so, because we run the tests in the root source dir. Probably we should refactor the tests to create a temp dir and run in there, then we could remove a lot of the .gitignore actually...